Jellyfin.Plugin.Intros/Configuration/config.html

```diff
@@ -14,23 +14,60 @@
+                            <option value="442744420">Audio Waveform</option>
+                            <option value="443076423">Blue Smoke</option>
+                            <option value="443381251">Blue Smoke Whirl</option>
                             <option value="442179153">Cinematic Explosion</option>
-                            <option value="442981865">Netflix</option>
+                            <option value="443404430">Classic Black</option>
+                            <option value="443082153">Color Wheel</option>
+                            <option value="443076524">Colorful Lights</option>
+                            <option value="443076432">Colorful Particles</option>
+                            <option value="443081703">Colorful Swirls</option>
                             <option value="443076518">Colorful Transformation</option>
-                            <option value="443076479">Neon Particles</option>
-                            <option value="443081835">Cyberpunk</option>
-                            <option value="443076530">X Logo</option>
-                            <option value="443404335">Neon Blue</option>
-                            <option value="443076423">Blue Smoke</option>
+                            <option value="441615803">COVID-19 Social Distancing</option>
+                            <option value="443081835">Cyberpunk Loading</option>
+                            <option value="443081756">Cyberpunk</option>
+                            <option value="443404417">Dark Matter</option>
+                            <option value="443081683">Demolition</option>
+                            <option value="443076427">Digital Glitch</option>
+                            <option value="443404395">Discowave</option>
                             <option value="443081932">Dubstep</option>
+                            <option value="440785270">DVD Coming Soon</option>
+                            <option value="440791181">DVD Feature Presentation</option>
                             <option value="443125346">Elegant Short</option>
-                            <option value="440791181">Retro DVD Intro</option>
-                            <option value="440793415">Alamo Drafthouse No Talking</option>
-                            <option value="440978850">Alamo Drafthouse Shush</option>
-                            <option value="442118203">IT Crowd Anti-Piracy</option>
+                            <option value="443076495">Energy Glitch</option>
+                            <option value="443082249">Fire Explosion</option>
+                            <option value="443381171">Glitch Freeloader</option>
+                            <option value="443381220">Glitch</option>
+                            <option value="443076410">Google Search (Clean)</option>
+                            <option value="443076406">Google Search (Explicit)</option>

```

Comment:
```suggestion
                            <option value="443076410">Google Search Clean</option>
                            <option value="443076406">Google Search Explicit</option>
```

Reply:
@njchaltry maybe label one of them V2 or use a different name when the labels are moved to tags. The clean one could be `Stay the Heck Home` or something.

Reply:
Easy enough. You only depend on the Vimeo ID right? So, updating titles or adding tags shouldn't affect anything?

Reply:
Yeah, the only thing I use right now is the IDs so metadata updates won't affect anything.

Reply:
@dkanada Vimeo API actually supports [content ratings](https://developer.vimeo.com/api/reference/responses/content-rating) so I'll work on marking violence and profanity as mature.

Reply:
changed to match vimeo titles